src/pages/iou/IOUModal.js

```diff
@@ -276,6 +285,21 @@ class IOUModal extends Component {
             return;
         }
 
+        if (this.isSendRequest) {
+            IOU.payIOUReport({
+                chatReportID: this.props.route.params.reportID,
+                reportID: 0,
+                paymentMethodType: paymentMethod,
+                amount: Math.round(this.state.amount * 100),
+                currency: this.props.iou.selectedCurrencyCode,
+                submitterPayPalMeAddress: this.state.participants[0].payPalMeAddress,
+                submitterPhoneNumber: this.state.participants[0].phoneNumber,
+                comment: this.state.comment,
+                requestorEmail: this.state.participants[0].login,
```

Comment:
NAB, guessing that `requestor` and `submitter` refer to the same person since we are looking at `participants[0]`. If that's correct, we probably should pick one to use and stick with it.